1-exercises/A-accessing-values/exercise3.js

```diff
@@ -21,7 +21,8 @@ let basketballTeam = {
 */
 
 // write code here
-
+  topPlayers=basketballTeam.topPlayers.sort().join("\n")
+  console.log(topPlayers)
 
```

Comment:
good job Bushra :), I think here we wanted you to show each top player, instead of showing the array :) , Like this : 
`basketballTeam.topPlayers.sort().forEach/map(player => console.log(player))`